src/resmom/start_exec.c

```diff
@@ -2125,6 +2135,9 @@ finish_exec(job *pjob)
 	int			downfds2 = -1;	/* init to invalid fd */
 	int			port_out, port_err;
 	struct startjob_rtn	sjr;
+#if MOM_ALPS
+	struct startjob_rtn     ack;
```

Comment:
What is ack used for, other than reading from the pipe? We assign it a value, but never read it.

Reply:
I wanted to accomplish what the starter_return() function does but without closing the fds. So I followed the same style as that of starter_return()

Reply:
Looking at it now, I see that it blocks until the ack comes in. There's nothing wrong with the code.